src/appleseed/foundation/utility/api/apistring.cpp

```diff
@@ -69,7 +76,13 @@ APIString& APIString::operator=(const APIString& rhs)
     return *this;
 }
 
-bool APIString::operator==(const APIString& rhs) const
+APIString& APIString::operator=(APIString&& rhs) APPLESEED_NOEXCEPT
+{
+    swap(m_s, rhs.m_s);
```

Comment:
Are you sure this is correct? Shouldn't you simply set `rhs.m_s` to nullptr like you did in the move constructor?

Reply:
This is correct, because rhs is a temporary and will be destroyed after the function returns.
If you swap the remote parts, rhs automatically deletes the previous remote part of this object.